nucypher/blockchain/eth/multisig.py

```diff
@@ -93,7 +94,7 @@ def decode_transaction_data(self, contract: Contract = None, registry=None) -> T
         else:
             raise ValueError("This proposed TX doesn't have data")
 
-    def write(self, filepath: str = None) -> str:
+    def write(self, filepath: Path = None) -> Path:
```

Comment:
`Optional`?

Reply:
I think `filepath` shouldn't ever be `None` - otherwise it will throw at `open(filepath)`.